apps/contacts/resource.py

```diff
@@ -81,7 +81,17 @@ class ContactsResource(Resource):
         "city": {"type": "string", "required": False},
         "contact_state": {"type": "string", "required": False},
         "postcode": {"type": "string", "required": False},
-        "country": {"type": "string", "required": False},
+        "country": {
+            "type": "dict",
+            "schema": {
+                "name": {"type": "string", "required": False},
+                "qcode": {"type": "string", "required": False},
+                "translations": {
+                    "type": "dict",
```

Comment:
would it work without schema? maybe you could add/update a test for this

Reply:
yes, it would work, but I thought it would be good if we define the schema for making it specific. what's your suggestion here, should I remove it and make it just as `"country": {"type": "dict", "required": False}`

Reply:
I mean the schema for `translations`

Reply:
Ah ok, yes it's working properly without schema.